nbstripout.py

```diff
@@ -112,6 +112,10 @@ def strip_output(nb):
     """strip the outputs from a notebook object"""
     nb.metadata.pop('signature', None)
     for cell in _cells(nb):
+        if (cell.metadata.get('init_cell', False) or
```

Comment:
Passing `False` to `get` is redundant, since the default return value for not found keys is `None`, which evaluates to `False`.